src/main/java/com/deque/axe/android/AxeView.java

```diff
@@ -191,7 +191,7 @@ public String speakableTextRecursive() {
     return allAreNull.get() ? null : result.toString();
   }
 
-  public String speakableText() {
+  private String speakableText() {
```

Comment:
Does this change line 24 in `ActiveViewName`?

Reply:
This was an auto-ide fix. It should have added an "allow this to be public even though nobody in this repo accesses it" annotation.